library/core/src/task/wake.rs

```diff
@@ -336,8 +337,8 @@ pub struct ContextBuilder<'a> {
 impl<'a> ContextBuilder<'a> {
     /// Create a ContextBuilder from a Waker.
     #[inline]
-    #[rustc_const_unstable(feature = "const_waker", issue = "102012")]
     #[unstable(feature = "local_waker", issue = "118959")]
+    #[rustc_const_unstable(feature = "const_local_waker", issue = "118959")]
     pub const fn from_waker(waker: &'a Waker) -> Self {
```

Comment:
This is your CI failure; because it is marked `const`-unstable, it can't be called from other const functions.

You should be able to make it `rustc_const_stable` without actually publicly stabilizing it, since this seems low risk. I think you need to use `const_waker` for the feature if making it `rustc_const_stable`, otherwise it will probably complain that `const_local_waker` is both stable and stable.

Reply:
I think `#[rustc_allow_const_fn_unstable]` is the right way to go.